src/main/java/com/nylas/NeuralSignatureContacts.java

```diff
@@ -0,0 +1,117 @@
+package com.nylas;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class NeuralSignatureContacts {
+	private List<String> job_titles;
+	private List<Link> links;
+	private List<Name> names;
+	private List<String> phone_numbers;
+	private List<String> emails;
+
+	public List<String> getJobTitles() {
+		return job_titles;
+	}
+
+	public List<Link> getLinks() {
+		return links;
+	}
+
+	public List<Name> getNames() {
+		return names;
+	}
+
+	public List<String> getPhoneNumbers() {
+		return phone_numbers;
+	}
+
+	public List<String> getEmails() {
+		return emails;
+	}
+
+	/**
+	 * Creates a Nylas contact object compatible with the contact endpoints.
+	 * Please note if multiple names were parsed only the first set are used.
```

Comment:
can we amend the comment to mention job titles as well - i.e. `multiple names or job titles`?